test/date.js

```diff
@@ -153,25 +153,29 @@ describe('Date Component', () => {
           'date-field': error
         };
         date.hooks['pre-render'](req, res, next);
-        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { error, key: 'date-field', legend: null, legendClassName: '', hint: null });
+        expect(res.render).to.have.been.calledWith(sinon.match.string, sinon.match({error}));
```

Comment:
hahaaa
actually I had it that way first but I wasn't sure what the team prefer? couldn't the linting pick this up?
thanks for merging still